### ISSUE TITLE
Markdown table rendering problem

### ISSUE THREAD

**Issue by Cli4d** [OpenTermsArchive/engine#1119]
### Overview
The rendering of the markdown table syntax for a declaration being tracked from [PR 814](https://github.com/OpenTermsArchive/france-declarations/pull/814) in France Collections does not render well, making the generated version unreadable.

Here is a screenshot of the generated version section as seen from the contribution tool

<img width="607" alt="chrome_PCinapMBgd" src="https://github.com/user-attachments/assets/036aa284-859f-4f66-9862-02aaeeef15f6">

You can check the [generated version](url) as well

**Comment by Cli4d** [comment]
It seems the generated version has no errors as viewed in the contribution tool earlier. Could it be the contribution tool provided a false alert?

**Comment by MattiSG** [comment]
Yes, it is definitely possible that the contribution tool has a different Markdown renderer than GitHub's 🙂 This issue could thus be moved to https://github.com/OpenTermsArchive/contribution-tool/

**Comment by Cli4d** [comment]
Ok. However, I do not have permission to transfer issues from this repository. I don't have the transfer issue option in my right sidebar for the issue.
